cyclus/main.py

```diff
@@ -314,7 +314,7 @@ def make_parser():
                    help='restart from the specified simulation snapshot, '
                         'not supported.')
     p.add_argument('--schema', action=Schema,
-                   help='dump the cyclus master schema including all '
+                   help='dump the cyclus main schema including all '
```

Comment:
Similar comment as the other file about changing the terms for the schema

Reply:
as above - this is a generic adjective word that I thought was OK